actors/reward/tests/reward_actor_test.rs

```diff
@@ -146,46 +146,59 @@ mod test_award_block_reward {
     }
 
     #[test]
-    // TODO remove ignore when fixing (v0->v2 migration)
-    #[ignore = "invalidated -- update"]
-    fn pays_reward_and_burns_penalty() {
+    fn pays_reward_and_tracks_penalty() {
         let mut rt = construct_and_verify(&StoragePower::default());
         rt.set_balance(TokenAmount::from(10_i128.pow(27)));
         rt.expect_validate_caller_addr(vec![*SYSTEM_ACTOR_ADDR]);
         let penalty: TokenAmount = TokenAmount::from(100);
         let gas_reward: TokenAmount = TokenAmount::from(200);
-        let expected_reward = &*EPOCH_ZERO_REWARD / 5 + &gas_reward - &penalty;
-        assert!(
-            award_block_reward(&mut rt, *WINNER, penalty, gas_reward, 1, expected_reward).is_ok()
+        let expected_reward: TokenAmount = &*EPOCH_ZERO_REWARD / 5 + &gas_reward;
```

Comment:
to improve on golang test this magic numbers should be replaced with the expected wins per epoch parameter

Reply:
@ZenGround0  What expected wins per epoch parameter?

Reply:
`EXPECTED_LEADERS_PER_EPOCH`

Reply:
replaced magic number with `EXPECTED_LEADERS_PER_EPOCH`

Reply:
thx!